3.10/indexing-persistent.md

```diff
@@ -25,11 +34,16 @@ If the index is declared sparse, a document will be excluded from the index and
 uniqueness checks will be performed if any index attribute value is not set or has a value 
 of `null`. 
 
-It is not possible to create multiple persistent indexes with the same `fields` attributes
-and uniqueness but different `storedValues` attributes. That means the value of 
-`storedValues` is not considered in index creation calls when checking if a persistent
-index is already present or needs to be created.
+Since ArangoDB 3.10, an in-memory hash cache can optionally be put in front of persistent
```

Comment:
I always have mixed feelings about "since <version>" clauses. They don't age well.
Also: https://developers.google.com/style/timeless-documentation

Reply:
According to the "timeless documentation" page you linked, the usage of an exact version number should be fine:

> Words and phrases that assume prior knowledge of a product or feature. If you must use words like new, give a reference point such as a date or version release number—for example, The January 14, 2021 release of BigQuery includes a new resource panel.

But I am also happy to remove again references to versions here. I put them in only because the documentation stakeholders wanted that in the past. I am more than happy to give up on adding version hints here.